src/test_typing_extensions.py

```diff
@@ -4728,8 +4783,8 @@ class ExtraReadOnly(TypedDict):
         self.assertEqual(ExtraReadOnly.__optional_keys__, frozenset({}))
         self.assertEqual(ExtraReadOnly.__readonly_keys__, frozenset({'__extra_items__'}))
         self.assertEqual(ExtraReadOnly.__mutable_keys__, frozenset({}))
-        self.assertEqual(ExtraReadOnly.__extra_items__, None)
-        self.assertFalse(ExtraReadOnly.__closed__)
+        self.assertIs(ExtraReadOnly.__extra_items__, NoExtraItems)
```

Comment:
maybe change this test name to include legacy? kind of reads confusingly otherwise

Reply:
Good point, done